src/config/dbConfig.js

```diff
@@ -0,0 +1,6 @@
+import Sequelize from "sequelize";
+import dotenv from "dotenv";
+
+dotenv.config();
```

Comment:
There is already a separate task for integrating dotenv package.

Reply:
I'm sorry I didn't think of alternatives at a time, I should have used plain URL for testing.